src/main/resources/templates/pebt/hasIncome.html

```diff
@@ -1,4 +1,4 @@
-<th:block th:with="messageVariant=${inputData.hasHousehold == 'true' ? 'multiple' : 'individual'}">
+<th:block th:with="messageVariant=${inputData.getOrDefault('isApplyingForSelf', 'false') == 'true' ? 'individual' : 'multiple'}">
```

Comment:
Renaming `individual` to `self` seems like a win. Maybe renaming `multiple` is fine, hard to say.